netcode.c

```diff
@@ -5415,8 +5426,8 @@ static void test_challenge_token()
     // encrypt the buffer
 
     uint64_t sequence = 1000;
-    uint8_t key[NETCODE_KEY_BYTES]; 
-    netcode_generate_key( key );    
```

Comment:
SO I went through your commits to try to find where this file got reformatted and looks like maybe `Connect token encryption/decryption passing with match_id.` is the winner.... Do you know if that commit is revert-able?  It's going to be hard to review - I can `&w=1` to ignore the whitespace changes but can't comment in that mode.

When forking 3rd party code, it's important to have a history of changes in case we ever have to upgrade.

Reply:
Also it's good that you're so consistent about commits - I usually just fall back to local history in Intellij.